.github/workflows/deploy-to-control-plane.yml

```
@@ -0,0 +1,229 @@
+              body: '🚀 Deploying to Control Plane...\n\n[View Deployment Progress](' + jobUrl + ')'
+            });
+            
+            return {
+              deploymentId: deployment.data.id,
+              commentId: comment.data.id,
+              workflowUrl: getWorkflowUrl(context.runId)
+            };
+
+      - name: Build Docker Image
+        if: steps.determine_action.outputs.action == 'deploy'
+        uses: ./.github/actions/build-docker-image
```

actionlint (1.7.4)

missing input "org" which is required by action "Build Docker Image" defined at "./.github/actions/build-docker-image". all required inputs are "app_name", "commit", "org"

(action)